dev/changelog/48.0.1.md

```diff
@@ -0,0 +1,41 @@
+<!--
```

Comment:
Curious, why we are missing this doc.

Reply:
Nice catch! It is removed in upstream DF. See https://github.com/apache/datafusion/tree/branch-51/dev/changelog